roles/openshift_control_plane/tasks/encrypting_config.yml

```diff
@@ -0,0 +1,54 @@
+---
+- when: openshift_encryption_config is defined
+  block:
+  - name: check if there are existant config
+    stat:
+      path: "{{ openshift.common.config_base }}/master/encryption-config.yaml"
+    register: stat_encrypting_file
+
+  - name: create tempfile to store new config
+    tempfile:
+      state: file
+      suffix: ansible
+    register: temp_file
+
+  - name: copy new config to temp file
+    copy:
+      content: "{{ openshift_encryption_config }}"
+      dest: "{{ temp_file.path }}"
+
+  - name: stat the temporary file
+    stat:
+      path: "{{ temp_file.path }}"
+    register: stat_new_encrypting_config
+
+  - name: Generate encryption config
+    copy:
+      content: "{{ openshift_encryption_config }}"
+      dest: "{{ openshift.common.config_base }}/master/encryption-config.yaml"
+    when: not stat_encrypting_file.stat.exists or stat_encrypting_file.stat.md5 != stat_new_encrypting_config.stat.md5
```

Comment:
We don't need to check for any of this stuff.  This should only affect new installs, just copy the data into place.